src/localstack.js

```diff
@@ -23,19 +24,18 @@ class TempWriteBuffer extends Writable {
   }
 
   toString (encoding) {
-    return this._buffer.map((chunk) => chunk.toString(encoding)).join('');
+    return this._buffer.join('\n');
   }
 
   _write (chunk, encoding, callback) {
     const asBuffer = Buffer.from(chunk, 'utf8');
-    const asString = asBuffer.toString('utf8');
-    if (this._buffer) {
+    const asString = asBuffer.toString('utf8').replace(/\r/, '').trim();
+    const logs = asString.split('\n');
+    this._buffer.push(...logs);
+    if (!this._isSetUp) {
       this.logger.debug(asString);
-      this._buffer.push(asBuffer);
-      const logs = this.toString('utf8').trim().split('\n');
```

Comment:
Similar to above - I think this was the only usage of the function. Can it be deleted?

Reply:
I was actually thinking about moving it out to be exposed in the test context, then results from Lambda could be exposed like the current docker lambda instance.